demo/cache/cache-controller.js

```diff
@@ -2,49 +2,31 @@ app.controller('CacheCtrl', function($scope, $rootScope, $http, $cookies, File,
 		Caching, Log) {    
 	$scope.isCachingStarted = false;
 	$scope.fairDataPoints = [];
+	$scope.fdp = {'url':"", 'name':"", "state":true};
```

Comment:
You're mixing `'` and `"` quotation marks for the keys. Either use the same quotation marks throughout your code, or (in this case) you can omit the quotation marks for your keys: `{url: "", name: "", state: true}` is valid javascript.